src/System.Text.Json/src/System/Text/Json/Serialization/Converters/JsonValueConverterUri.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json.Serialization.Policies;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal sealed class JsonValueConverterUri : JsonValueConverter<Uri>
```

Comment:
Can you hold off from committing this until https://github.com/dotnet/corefx/pull/38578 is in? There will be some minor refactoring. Thanks